MRN-Code/coinstac
# Redesign/consortium save fix

* **Problem:**
   1. Consortium items were able to be created but not edited
* **Solution:**
   1. Added code in resolvers.js to save consortium item from rethinkdb to both accurately save information as well as updated the users tables based on permissions for the new consortium. This allows the user to later edit and save changes
* **Tests:**
   1. Click on consortia tab on the left
   1. Create new consortium or click on view details of an already made consortium
   1. Make any changes you desire to name and description and hit save
   1. Navigate back to consortium tab and see updated consortium name/description
